editor/karma-base.conf.js

```diff
@@ -1,4 +1,6 @@
-process.env.CHROME_BIN = require('puppeteer').executablePath() // Puppeteer v19.6.0 uses Chromium 110.0.5479.0
+if (!('CHROME_BIN' in process.env)) {
+  process.env.CHROME_BIN = require('puppeteer').executablePath() // Puppeteer v19.6.0 uses Chromium 110.0.5479.0
+}
```

Comment:
Does this mean we will be able to run the tests with an arbitrary Chrome executable? Why should we have this rather than a fixed Chrome version?

Reply:
Yes, with the default behaviour being to use the fixed Chrome version supplied by Puppeteer.

The reason why this is beneficial is a long standing issue of npm packages that assume the particular binary they've bundled will work on every variant of the platform. In the case of Linux that's a big assumption as it usually means the bundle expects things like glibc to live in one particular directory, which is not always the case and results in Chrome insta-crashing.